jenkins/spark-nightly-build.sh

```diff
@@ -24,29 +24,35 @@ export M2DIR="$WORKSPACE/.m2"
 # build, install, and deploy all the versions we support, but skip deploy of individual dist module since we
 # only want the combined jar to be pushed.
 # Note this does not run any integration tests
+# Deploy jars unless SKIP_DEPLOY is 'true'
+SKIP_DEPLOY=${SKIP_DEPLOY:-'false'}
```

Comment:
nit: technically unnecessary after the condition was reworked to `[[ $SKIP_DEPLOY != 'true' ]]`

Reply:
fixed